.gitignore

```diff
@@ -69,3 +69,4 @@ Thumbs.db
 *.bin
 *.run
 .qmake.conf
+*.res
```

Comment:
What's this for?

Reply:
don't you have many .res files generated by msvc?

Reply:
MSVC will generate many .res files indeed, but they won't pullote your source code dir if you are doing shadow builds (out-of-source builds). Qt Creator will always do shadow builds by default. These files will be in a separate dir. Maybe you are doing in-source builds, and thus you have these files in the source directory.

Reply:
could be, is that a problem to have this in the .ignore file?

Reply:
No, just point this out. I'm fine with it.